<?xml version="1.0"?>
<launch>

    <node name="gps_converter" pkg="brave_2020_localisation" type="gps_converter.py" output='screen' />

<!--    You can indicate here the coordinates of the different buoys, format: "['48.198924, -3.016511', '48.186924, -3.013811']" -->
<!--  	Leaving it empty will ask you later for the coordinates in a more interactive way. -->
    <arg name="marks_coordinates" default="[]" />
    <param name="marks_coordinates" type="str" value="$(arg marks_coordinates)" />

</launch>